File: 200_LeetCode_Questions/C++/D04P3_LeetCode_1450.cpp
// Number of Students Doing Homework at a Given Time

//Given two integer arrays startTime and endTime and given an integer queryTime.
//The ith student started doing their homework at the time startTime[i] and finished it at time endTime[i].
//Return the number of students doing their homework at time queryTime. More formally, 
//return the number of students where queryTime lays in the interval [startTime[i], endTime[i]] inclusive.

class Solution
{
public:
    int busyStudent(vector<int>& startTime, vector<int>& endTime, int queryTime)
	{
        int cnt = 0,i;
        for (i = 0; i < startTime.size(); i++)
		{
            if (queryTime <= endTime[i] && queryTime >= startTime[i])
			{
                cnt++;
            }
        }
        return cnt;
    }
};

File: 200_LeetCode_Questions/Python/D04P3_LeetCode_1450.py
class Solution:
    def busyStudent(self, startTime: List[int], endTime: List[int], queryTime: int) -> int:
        count=0
        j=0
        while j<len(startTime):
            for i in range(startTime[j],endTime[j]+1):
                if i==queryTime:
                    count+=1
            j+=1
        return count
